ext/libclementine-tagreader/tagreader.cpp

```diff
@@ -130,6 +130,51 @@ const char* kASF_OriginalDate_ID = "WM/OriginalReleaseTime";
 const char* kASF_OriginalYear_ID = "WM/OriginalReleaseYear";
 }  // namespace
 
+void TagReader::RemoveExtension(std::string* s) const {
+  if (s->empty()) return;
+  const int i = s->find_last_of('.');
+  if (i == std::string::npos) return;
+  s->assign(s->substr(0, i));
+}
+
+void TagReader::ChangeUnderscores(std::string* s) const {
+  for (int i = 0; i < s->size(); i++) {
+    if (s->at(i) == '_') s->at(i) = ' ';
+  }
+}
+
+std::string& TagReader::ltrim(std::string& str, const std::string& chars) const {
```

Comment:
We only use pointers, const references and values to keep things readable: https://google.github.io/styleguide/cppguide.html